components/dialog/index.css

```
@@ -113,13 +101,13 @@ governing permissions and limitations under the License.
 
   margin: 0;
 
-  font-size: var(--spectrum-dialog-confirm-title-text-size);
-  font-weight: var(--spectrum-dialog-confirm-title-text-font-weight);
-  line-height: var(--spectrum-dialog-confirm-title-text-line-height);
-
+  font-size: var(--mod-dialog-confirm-title-text-size,var(--spectrum-dialog-confirm-title-text-size));
+  font-weight: var(--mod-dialog-confirm-title-text-font-weight,var(-spectrum-dialog-confirm-title-text-font-weight));
```

```suggestion
  font-weight: var(--mod-dialog-confirm-title-text-font-weight,var(--spectrum-dialog-confirm-title-text-font-weight));
```

One last visual thing, just a small typo here that is causing the font-weight for Express theme to be 700 instead of 900